articles/concepts-multiple-qubits.md

```diff
@@ -78,15 +82,15 @@ $$
     \end{bmatrix}.
 $$
 
-It is also possible to measure just one qubit of a two-qubit quantum state. In cases where you measure only one of the qubits, the impact of measurement is subtly different because the entire state is not collapsed to a computational basis state, rather it is collapsed to only one subsystem.  In other words, in such cases measuring only one qubit only collapses one of the subsystems but not all of them.  
+It is also possible to measure just one qubit of a two-qubit quantum state. In such case, the impact is subtly different; only the related subsystem, not the entire state, is collapsed to a computational basis state.  
```

Comment:
```suggestion
It's also possible to measure just one qubit of a two-qubit quantum state. When you measure only one qubit of a two-qubit state, the impact of measurement is subtly different than measuring two qubits. This is because the entire state isn't collapsed to a computational basis state, rather it's collapsed to only one subsystem.  In other words, measuring one qubit of a two-qubit state only collapses the related subsystem to a computational basis state. 
```

Reply:
@SoniaLopezBravo I seriously don't understand from the context what exactly you mean by
> This is because the entire state isn't collapsed to a computational basis state, rather it's collapsed to only one subsystem.

But intuitively it sounds wrong to me. I would say "only one subsystem of the whole system gets collapsed", not "the whole system gets collapsed to only one subsystem". And the next sentence supports that.

And do you think that duplicated phrases like
> It's also possible to measure just **one qubit of a two-qubit quantum state**. When you measure only **one qubit of a two-qubit state** ...

are what makes the text more readable?

I though pull requests could be more about discussion to find a clear formulation. If I'm suggesting a change, it's because I as a new reader was disturbed by something in the formulation or had issues understanding it.